### PR TITLE
Smaller timer value for Python 3 on Windows

### PR DESCRIPTION
**Pull request by SaschaNaz**
Python 3 limits timeout value to [`threading.TIMEOUT_MAX`](https://docs.python.org/3/library/threading.html#threading.TIMEOUT_MAX) (4294967.0 on Windows), so current `sys.maxint` causes *OverflowError: timeout value is too large* when generating system libs on Windows.

Unfortunately Python 2 does not provide `threading.TIMEOUT_MAX` so this PR reduces the timeout value to 999999 seconds (about 11 days) which should be reasonable enough.

### CODE REVIEW

File: tools/system_libs.py [hunk 1]
```diff
@@ -26,7 +25,7 @@ def run_commands(commands):
   else:
     pool = shared.Building.get_multiprocessing_pool()
     # https://stackoverflow.com/questions/1408356/keyboard-interrupts-with-pythons-multiprocessing-pool, https://bugs.python.org/issue8296
-    pool.map_async(call_process, commands, chunksize=1).get(maxint)
```
Comment (kripken) [inline tools/system_libs.py]: please add a comment about the constant, that it is just something huge for several days and that works in both 2 and 3

**Comment by SaschaNaz** [comment]
Somehow Travis keeps failing to pull a Docker image :/

**Comment by SaschaNaz** [comment]
I didn't know that, thanks!

**Comment by SaschaNaz** [comment]
What's happening on the Travis side today, BTW.

**Pull request merged**